### PR TITLE
[onert-micro] Support SplitV kernel

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr adds SplitV kernel with unit test

for issue: #10709

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: onert-micro/luci-interpreter/src/kernels/Split.cpp [hunk 1]
```diff
@@ -16,67 +16,10 @@
...
-      const int copy_size = Tensor::dim(output0, axis) * base_inner_size;
-      T *output_ptr = output_data + k * copy_size;
-      for (int j = 0; j < copy_size; ++j)
-        output_ptr[j] = input_ptr[j];
-      input_ptr += copy_size;
-    }
-  }
-}
-
-} // namespace
```
Comment (BalyshevArtem) [inline onert-micro/luci-interpreter/src/kernels/Split.cpp:79]: move common part with SplitV into Split.h file

File: onert-micro/luci-interpreter/src/kernels/Split.h [hunk 2]
```diff
@@ -14,34 +14,65 @@
...
-};
+      T *output_data = kernels::getTensorData<T>(runtime_graph->getDataByTensor(output));
+      const int copy_size = Tensor::dim(output, axis_value) * base_inner_size;
+      T *output_ptr = output_data + k * copy_size;
+      for (int j = 0; j < copy_size; ++j)
+        output_ptr[j] = input_ptr[j];
+      input_ptr += copy_size;
+    }
+  }
+}
```
Comment (BalyshevArtem) [inline onert-micro/luci-interpreter/src/kernels/Split.h:74]: common part of Split and SplitV kernels

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Comment by BalyshevArtem** [comment]
added license line
@SlavikMIPT, @chunseoklee PTAL

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**